chapters/en/slides/module6/module6_02.Rmd

```diff
@@ -0,0 +1,352 @@
+{
+  "type": "Feature",
+  "id": 8,
+  "properties": {"name": "Colorado"},
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [[-106.32056285448942,40.998675790862656],[-106.19134826714341,40.99813863734313],
+       [-105.27607827344248,40.99813863734313],[-104.9422739227986,40.99813863734313],
+       [-104.05212898774828,41.00136155846029],[-103.57475287338661,41.00189871197981],
+       [-106.32056285448942,40.998675790862656]]
+    ]
+  }
+}
+```
+
+Notes:
+Up until now,
+all the data we have plotted in Altair have been read from CSV-file
+as tidy dataframes.
+To visualize data on maps we will be using
+file formats that are designed specifically for geographic data.
+
+These are called GeoJSON and TopoJSON
+and are saved as plain text file,
+but with rules for how store geographic data such as coordinates.
+You can see an example of how they look in this slide.
+
+---
+
+## Creating a map in Altair
+
+```{python}
+import altair as alt
+from vega_datasets import data
+
+world_map = alt.topo_feature(data.world_110m.url, 'countries')
+alt.Chart(world_map).mark_geoshape()
+```
+
+Notes:
+The vega sample data sets contain a TopoJSON dataset for the world and the US,
+so let's use that here.
+
+Altair work directly with GeoJSON files,
+so to use the TopoJSON sample data,
+we use the `alt.topo_feature` help function
+so that Altair can read the TopoJSON file correctly.
+
+We then use `mark_geoshape` to visualize this geographic TopoJSON data
+in the form of a map.
+Since the data contains the border coordinates of each country,
+we can see these drawn as white lines in the chart on this slide.
+
+But why does Antarctica look so much bigger
+than the rest of the world?
+
+---
+
+## Changing the projection gives a more accurate representation of areas
+
+```{python}
+alt.Chart(world_map).mark_geoshape().project(type='equalEarth')
+```
+
+Notes:
+Since the Earth is a sphere
+it is not entirely straightforward
+how to represent the areas of the different parts of the world
+on a 2D visualization.
+
+The reason Antarctica looked so giant in the previous slide
+is because of how the spheres is being stretched 
+when it is projected onto a 2D surface.
+
+The default projection method for maps is often `mercator`,
+but it causes heavy distortion of the relative areas of land masses,
+which get worse the closer to the poles we come.
+
+Fortunately,
+many people are moving away from using the projection
+because it gives an incorrect impression of how the world looks.
+A commonly used alternative is the `equalEarth` projection,
+which focuses on correctly representing the relative areas of all landmasses.
+
+Some of the most noticeable differences in this projection
+are that Africa appears much bigger
+and Greenland, Antarctica and Russia appear notably smaller.
+Africa is in fact 15x the area of Greenland,
+far from the `mercator` projection which shows them as roughly the same size.
```

Comment:
```suggestion
Given that the Earth is a sphere
it is not entirely straightforward
how to represent the areas of the different parts of the world
on a two dimensional visualization.

The reason that Antarctica looked so big in the previous slide
is because of how the sphere is being stretched 
when it is projected onto a two dimensional visualization surface.

`mercator` is often used as the default projection method for maps.
However, it causes heavy distortion of the relative areas of land masses,
and in particular, this distortion gets much worse closer to the poles.

Fortunately,
many people are moving away from using this projection
because it gives a distorted impression of what the world looks like.
A commonly used alternative is the `equalEarth` projection.
This projection focuses on correctly representing the relative areas of all landmasses.

Some of the most noticeable differences in this projection
are that Africa appears much bigger
and Greenland, Antarctica as well as Russia appear notably smaller.

Africa is in fact 15x the area of Greenland, this is far
far from how the `mercator` projection draws this - it draws them
 as roughly the same size.
```